portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro

```diff
@@ -816,6 +816,35 @@ definition {
 		BlogsEntry.viewStatus(entryStatus = "Approved");
 	}
 
+	@description = Create certificates again after upgrade
+	macro configureCertificatesAfterUpgrade {
```

Comment:
If you name this macro as "createCertificatedAfterUpgrade" then you can delete the @description part.

Reply:
Changed naming to your idea, the meaning of the macro is more clear with this choice